dhall/src/Dhall/Src.hs

```diff
@@ -1,6 +1,8 @@
 {-# LANGUAGE DeriveDataTypeable #-}
 {-# LANGUAGE DeriveGeneric      #-}
+{-# LANGUAGE DeriveLift         #-}
 {-# LANGUAGE OverloadedStrings  #-}
+{-# LANGUAGE TemplateHaskell    #-}
```

Comment:
Note that @vmchale at one point requested that Dhall not use `TemplateHaskell` or `QuasiQuotes` due to issues with cross-compilation.  See: https://github.com/dhall-lang/dhall-haskell/issues/89

@vmchale: Is this still a concern for you?

Reply:
Actually, even if it were, we could guard this behind a `cabal configure` flag so that users can compile out Template Haskell support if necessary.   But let's still confirm whether or not we even need to do so

Reply:
It wouldn't be hard to remove it from this file.

Reply:
Oh, if it's easy to do, then go ahead.  The real issue here is that I just don't really understand what are the exact requirements for Dhall to be easy-to-cross-compile.  For example, it's not clear to me if depending on `th-lift-instances` matters or not for the original use case @vmchale had in mind.

Reply:
This can actually get away with just `TemplateHaskellQuotes`, which is cross-compilation friendly.

Reply:
I've changed this to `TemplateHaskellQuotes`, I hadn't even heard of that extension!